ratelimit_test.go

```diff
@@ -152,6 +169,7 @@ func TestUnlimited(t *testing.T) {
 }
 
 func TestRateLimiter(t *testing.T) {
+	t.Parallel()
```

Comment:
Let's do these separately: https://github.com/uber-go/ratelimit/pull/98

Reply:
This test approach is slower than the previous and takes 55 seconds without parallelization. That is why I added it to this PR.

Reply:
Sure, reasonable.

But, since (1) it's reasonable in all cases (2) it will show us the actual slow down of the new approach, let's push it separately?

Reply:
Yup, no problem